### PR TITLE
adding ip address endpoint and test

### PR DESCRIPTION
**Pull request by amanda103**
didn't mean to include the version change/changelog.md

### CODE REVIEW

File: lib/nylas/account.rb [hunk 1]
```diff
@@ -33,6 +33,11 @@ def revoke_all(keep_access_token: nil)
       response[:success]
     end
 
+    def ip_addresses
+        response = execute(method :get, path "#{resource_path}/ip_addresses")
```
Comment (HayleyCAnderson) [inline lib/nylas/account.rb]: Looks like you're missing colons here

File: lib/nylas/account.rb [hunk 2]
```diff
@@ -33,6 +33,11 @@ def revoke_all(keep_access_token: nil)
       response[:success]
     end
 
+    def ip_addresses
```
Comment (HayleyCAnderson) [inline lib/nylas/account.rb]: What are you expecting to be returned from this method?

File: lib/nylas/api.rb [hunk 3]
```diff
@@ -99,6 +99,13 @@ def revoke(access_token)
       response.code == 200 && response.empty?
     end
 
+    # Returns list of IP addresses
+    # @return [Boolean]
```
Comment (HayleyCAnderson) [inline lib/nylas/api.rb]: What are you expecting to be returned here? This says both a boolean and a list of ip addresses

File: lib/nylas/version.rb [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Nylas
-  VERSION = "4.2.4"
+  VERSION = "4.3.0"
```
Comment (HayleyCAnderson) [inline lib/nylas/version.rb]: Lmk if you need help removing these changes from this PR

File: lib/nylas/account.rb (hunk 2 above)
Comment (amanda103) [reply lib/nylas/account.rb]: something that looks like this {"updated_at": timestamp, "ip_addresses": [1234, 3457, 2213]}

Comment (amanda103) [reply lib/nylas/account.rb]: what is the correct variable name for that?

Comment (HayleyCAnderson) [reply lib/nylas/account.rb]: Hm okay. Yeah so this is getting into the problem with how this gem was written. I think you're going to have to make more, much less obvious changes including making a new model. We might have to pair on it.

File: lib/nylas/api.rb (hunk 3 above)
Comment (HayleyCAnderson) [reply lib/nylas/api.rb]: If you look above at this `@labels ||= Collection.new(model: Label, api: self)` type situation a few lines above, I think that's basically what you're going to want to do here. So you'll have to start with making an `IPAddress` model. Maybe see if you feel comfortable trying to follow that path and then lmk if you want to pair.

Comment (amanda103) [reply lib/nylas/api.rb]: Sounds good - I'll work on this tomorrow and ping you if I get stuck.

**Comment by amanda103** [comment]
I think I was confused on what tests looked like in ruby. Can you point me to where those tests would go for this endpoint?

**Pull request closed without merging**